src/store/reducers/index.ts

```diff
@@ -0,0 +1,34 @@
+//#region > Imports
+//> Redux
+// Allows to React components read data from a Redux store, and dispatch actions
+// to the store to update data.
+import { combineReducers } from "redux";
+// LoadingBar
+import { loadingBarReducer } from "react-redux-loading-bar";
+
+//> Reducers
+// User
+import userReducers from "./userReducers";
+import ohrwurmReducer from "./ohrwurmReducers";
+//#endregion
+
+//#region > Config
+const rootReducer = combineReducers({
+  // Loading bar
+  loadingBar: loadingBarReducer,
+  // User data
+  user: userReducers,
+  // Ohrwurm
+  ohrwurm: ohrwurmReducer,
+});
+//#endregion
+
+//#region > Exports
+export type RootState = ReturnType<typeof rootReducer>;
```

Comment:
Move to types file?

Reply:
RootState should be part of reducers/index.ts